### ISSUE TITLE
Migration to Scala 2.12

### ISSUE THREAD

**Issue by DanielaSfregola** [DanielaSfregola/twitter4s#54]
...we should support Scala 2.12!

**Comment by maciejbak85** [comment]
first needed akka http :)

**Comment by DanielaSfregola** [comment]
Indeed! I am working on it, you can see my progress on #55_migration_to_akka_http...hopefully these christmas days will be productive enough to get it done -- it is also blocking some other tickets.

As usual, contributions are more than welcome! :dancing_men:
